package.json

```diff
@@ -9,6 +9,7 @@
     "scripts": {
         "clean": "rimraf dist && npm un -g",
         "compile": "tsc",
+        "lint": "tslint -c tslint.json src/**/*.ts",
```

Comment:
Nice! can you add this lint into the build pipeline also?

Reply:
sure